ibis_substrait/compiler/translate.py

```diff
@@ -583,6 +583,36 @@ def unbound_table(
     )
 
 
+@translate.register(ops.LocalTable)
```

Comment:
`ibis-substrait` currently depends on ibis 2.x, so this is a non-starter as is.

I would suggest implementing `LocalTable` in `ibis-subtrait` and not in `ibis`. We can always push it back into `ibis` upstream once the implementation is fleshed out.

Reply:
OK. Should I cancel this PR and open a new one in Ibis? Or should we flush out the remaining issues here first?

Reply:
Let's finish up the existing comments and then move it out.